platform/vs/docker-sonic-vs/database_config.json

```diff
@@ -77,6 +82,11 @@
             "id" : 14,
             "separator": ":",
             "instance" : "redis"
+        },
+        "BMP_STATE_DB" : {
+            "id" : 20,
```

Comment:
Why is new database id 20 vs next in sequence 15?

Reply:
removed unnecessary change